Vaida

VaidaLinkuviene opened the pull request:
I changed the main image in sightseeing.html. It is not the same as in index.html. I do not know if it is ok. Let me know if you think that it is better to keep the same image in all html files. Btw, check if main dropdown works in index.html. Because it does not work for me, but it works in sightseeing.html (even if I did copy/paste from index.html).

VaidaLinkuviene committed:
Sightseing tours part without descriptions

VaidaLinkuviene committed:
Sightseeing part

ieva-abola reviewed (changes requested):
I like how it looks! I also like the idea of a different photo in every page, I also did that for Blog page. Maybe try other picture though, this one looks a bit pixelated. And maybe we can make it 50% of the page and without the main title? Similar to Lonely Planet Destinations page.

VaidaLinkuviene committed:
Start doing adventure and Leisure tours

VaidaLinkuviene committed:
sightseeing, adventure, leisure tours parts without header

VaidaLinkuviene committed:
Merge branch 'test' into vaida

VaidaLinkuviene committed:
Headers of sightseeing, leisure, adventure tours

VaidaLinkuviene committed:
Header images for sightseeing, leisure and adventure tour html files

ieva-abola reviewed (commented):
Great! Looking at the page makes me want to travel :D

-  It looks that in merging process the styles for index.html Blog section have disappeared. You can find them in test branch under "Blog section" comment in styles.css. 
- I can take care of the navbar buttons and all the functionality for all pages when merge is done ! 
- Need to fix what happens on the main page when you click on the Kayaking in laggon, Leisure in Porto, Hiking in Tatras. Also when choosing the destination. 
- Some of the navbar buttons are not quite visible indeed. Do we change the button styling for those separate pages or maybe background picture so they are more visible (also on hover)?

VaidaLinkuviene committed:
Changes according to review

VaidaLinkuviene commented:
Hi, I added styles for blog part; added navigation to pages  and changed images. Check if it looks ok now and double check if everything works ;) Thanks :) I want to travel now too :D
